src/components/status-tags-in-task-list-pages/_status-tags-in-task-list-pages.scss

```diff
@@ -16,8 +24,13 @@
 }
 
 .app-task-list__task-name {
+  display: block;
   width: auto;
 
+  @include govuk-media-query($from: 450px) {
+    float: left;
+  }
+
```

Comment:
As this is a change to the HMRC component, this shouldn't require an update to the Gov Frontend / GDS task list pattern css.

Reply:
I've updated commit. Please review